README.md

```diff
@@ -89,7 +91,7 @@ instead of passing this flag.
     # For programs which require a the Unix Domain Socket at a specific location, set an absolute path (overrides -dir):
     ./cloud_sql_proxy -dir=/cloudsql -instances=my-project:us-central1:sql-inst=unix:/my/custom/sql-socket &
     mysql -u root -S /my/custom/sql-socket
-    
+
```

Comment:
Is this my editor? Do you want me to revert this?

Reply:
Nah it's fine.